client/js/src/client.ts

```diff
@@ -358,9 +358,10 @@ export function api_factory(
 
 				const _config = await config_success(config);
 				// connect to the heartbeat endpoint via GET request
-				const heartbeat = new EventSource(
+				const heartbeat_url = new URL(
 					`${config.root}/heartbeat/${session_hash}`
 				);
+				const heartbeat = EventSource_factory(heartbeat_url);
```

Comment:
btw, this `heartbeat` variable is not used after the declaration here. Is it ok?

Reply:
I don't think we have to assign it to a variable since we just have to connect to it, we don't have to parse the messages on this channel.

Reply:
👍 
→ 26fd6da